dwi_ml/data/dataset/parameter_description.py

```diff
@@ -0,0 +1,45 @@
+PARAMETER_DESCRIPTION = """
```

Comment:
This looks specific for a task/project, right? May be it should be moved and renamed?

Reply:
Yes, this is meant to be temporary. Some parameter descriptions were repetitive between dataset, trainor, etc.

I just put this here to keep it while we modify the rest. We will add the descriptions as last step.